### DISCUSSION TITLE
3.1.4 several feature disable options in INSTALL.MD do not work

### DISCUSSION THREAD

**Discussion by JOULTICOA** [openssl/openssl#22510] [Q&A] [answered]
Hello,

The following options in INSTALL.MD do not seem to work:
no-quic
no-apps
no-docs
no-http

Perhaps they are depracated?

Additionally, is there a way to disable Shake?  i.e. a "no-shake" option?

Thank you

**Comment by paulidale** [comment]
Not supported in that version?

I know that some of these were added for and will be part of 3.2.

It's always sensible to use the installation and build instructions that come with the release you're building.

**Reply by JOULTICOA** [reply]
perhaps I using old tar.gz.  I will wait for 3.2 come out thank you

**Comment by mattcaswell** [comment, marked as answer]
All of these options were added in 3.2 and do not exist in the 3.1 version of INSTALL.md

**Comment by t8m** [comment]
> Additionally, is there a way to disable Shake? i.e. a "no-shake" option?

No, there is no way to do that.
